website/versioned_docs/version-2.0.0-alpha.72/guides/docs/sidebar.md

```diff
@@ -137,10 +138,11 @@ Example:
 {
   type: 'doc',
   id: 'doc1', // string - document id
+  label: 'Getting started' // Sidebar label text
 }
 ```
 
-Using just the [Document ID](#document-id) is also valid, the following is equivalent to the above:
+The sidebar_label in the markdown frontformatter has a higher precedence over the label key in `SidebarItemDoc`. Using just the [Document ID](#document-id) is also valid, the following is equivalent to the above:
```

Comment:
please don't backport this doc: this feature will only be available in the next release and won't work for older. docusaurus versions